src/Symfony/Component/HttpKernel/EventListener/ExceptionListener.php

```diff
@@ -30,11 +31,15 @@ class ExceptionListener implements EventSubscriberInterface
 {
     protected $controller;
     protected $logger;
+    protected $httpStatusCodeLogLevel = array();
```

Comment:
private?

Reply:
Is there a good reason? The rest of the class is protected/public I think because it is a commonly overridden class. So might be weird to have a single private method/variable here? Might want to look at the whole class to decide what the overridable parts should be?